src/commcare_cloud/ansible/roles/elasticsearch/templates/systemd/elasticsearch.service.j2

```diff
@@ -8,6 +8,9 @@ After=network-online.target
 Type=simple
 Environment=ES_HOME={{ elasticsearch_home }}
 Environment=ES_PATH_CONF={{ elasticsearch_conf_dir}}
+{% if elasticsearch_version == '6.8.23'%}
```

Comment:
Not super urgent but worth considering what behavior we think we want for higher versions. Right now it will just revert to the default java install, which 🤷. I wonder if the right answer is to just make it hard fail if the version is 7 or higher to force someone to decide what version of java they want or something like that. But if I were to guess, we probably want whatever java we've been using in which case we kind of want `>=` here. (Also `>=` doesn't semantically understand version strings so isn't 100% appropriate, but for the actual versions that will be encountered in this scenario, it will be right.)

Reply:
Yeah automatically failing to force us to use the bundled version of java on versions of ES 7 and up sounds good to me.